Wrappers/Python/test/test_functions.py

```diff
@@ -926,38 +926,51 @@ def test_Lipschitz4(self):
 
 class TestTotalVariation(unittest.TestCase):
 
+    def setUp(self) -> None:
+        self.tv = TotalVariation()
+        self.alpha = 0.15
+        self.tv_scaled = self.alpha * TotalVariation()
+        self.tv_iso = TotalVariation()
+        self.tv_aniso = TotalVariation(isotropic=False)
+        self.ig_real = ImageGeometry(3,4)
+        self.ig_complex = ImageGeometry(3,4, dtype=np.complex64)        
+        
     def test_regularisation_parameter(self):
-        tv = TotalVariation()
-        np.testing.assert_almost_equal(tv.regularisation_parameter, 1.)
+        np.testing.assert_almost_equal(self.tv.regularisation_parameter, 1.)
 
     def test_regularisation_parameter2(self):
-        alpha = 0.15
-        tv = alpha * TotalVariation()
-        np.testing.assert_almost_equal(tv.regularisation_parameter, alpha)
+        np.testing.assert_almost_equal(self.tv_scaled.regularisation_parameter, self.alpha)
     
     def test_rmul(self):
-        alpha = 0.15
-        tv = alpha * TotalVariation()
-        assert isinstance(tv, TotalVariation)
+        assert isinstance(self.tv_scaled, TotalVariation)
     
     def test_regularisation_parameter3(self):
-        tv = TotalVariation()
         try:
-            tv.regularisation_parameter = 'string'
+            self.tv.regularisation_parameter = 'string'
             assert False
         except TypeError as te:
-            print (te)
             assert True
     def test_rmul2(self):
         alpha = 'string'
         try:
             tv = alpha * TotalVariation()
             assert False
         except TypeError as te:
-            print (te)
             assert True
-    
 
+    def test_call_real(self):
```

Comment:
I think testing the isotropic and anisotropic results should be difference tests.

Reply:
Fixed